src/sentry/static/sentry/app/components/onboardingWizard/task.tsx

```diff
@@ -0,0 +1,287 @@
+      title={tct('[requisite] before completing this task', {
+        requisite: task.requisiteTasks[0].title,
+      })}
+    >
+      <TaskLockedIndicator />
+    </Tooltip>
+  );
+
+  const {SupplementComponent} = task;
+  const supplement = SupplementComponent && (
+    <SupplementComponent task={task} completeTask={() => onMarkComplete(task.task)} />
+  );
+
+  const skipAction = task.skippable && (
+    <SkipConfirm orgId={organization.slug} onSkip={handleSkip}>
+      {({skip}) => (
+        <SkipButton priority="link" onClick={skip}>
+          {t('Skip task')}
+        </SkipButton>
+      )}
+    </SkipConfirm>
+  );
+
+  return (
+    <Item interactive ref={forwardedRef} onClick={handleClick} data-test-id={task.task}>
+      <Title>
+        {IncompleteMarker}
+        {task.title}
+      </Title>
+      <Description>{`${task.description}. ${task.detailedDescription}`}</Description>
+      {task.requisiteTasks.length === 0 && (
+        <ActionBar>
+          {task.status === 'pending' ? (
+            <InProgressIndicator user={task.user} />
+          ) : (
+            <CTA>Setup now</CTA>
+          )}
+          {skipAction}
+          {supplement}
+        </ActionBar>
+      )}
+    </Item>
+  );
+}
+
+const Item = styled(Card)`
+  padding: ${space(3)};
+  position: relative;
+`;
+
+const Title = styled('h5')`
+  font-weight: normal;
+  display: grid;
+  grid-template-columns: max-content 1fr;
+  grid-gap: ${space(0.75)};
+  align-items: center;
+  margin: 0;
+`;
+
+const Description = styled('p')`
+  padding-top: ${space(1)};
+  font-size: ${p => p.theme.fontSizeSmall};
+  line-height: 1.75rem;
+  color: ${p => p.theme.gray3};
+  margin: 0;
+`;
+
+const ActionBar = styled('div')`
+  height: 40px;
+  border-top: 1px solid ${p => p.theme.borderLight};
+  margin: ${space(3)} -${space(3)} -${space(3)};
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 ${space(2)};
+`;
+
+const InProgressIndicator = styled(
+  ({user, ...p}: React.HTMLAttributes<HTMLDivElement> & {user?: AvatarUser | null}) => (
+    <div {...p}>
+      <Tooltip
+        disabled={!user}
+        containerDisplayMode="flex"
+        title={tct('This task has been started by [user]', {
+          user: user?.name,
+        })}
+      >
+        <IconEvent />
+      </Tooltip>
+      {t('Task in progress...')}
+    </div>
+  )
+)`
+  font-size: ${p => p.theme.fontSizeMedium};
+  font-weight: bold;
+  color: ${p => p.theme.yellowOrange};
+  display: grid;
+  grid-template-columns: max-content max-content;
+  align-items: center;
+  grid-gap: ${space(1)};
+`;
+
+const CTA = styled('div')`
+  color: ${p => p.theme.blue};
+  font-size: ${p => p.theme.fontSizeMedium};
+  font-weight: bold;
+`;
+
+const SkipButton = styled(Button)`
+  color: ${p => p.theme.gray2};
+`;
+
+const TaskLockedIndicator = styled(IconLock)``;
+
+TaskLockedIndicator.defaultProps = {
+  size: 'xs',
+  color: theme.redLight,
+};
```

Comment:
Does this need to be a styled component? It doesn't seem to be re-used anywhere

Reply:
Removed.